logs.go

```diff
@@ -91,6 +92,23 @@ func getLoggableResources(getPodContainers func(string, string) ([]string, error
 	return loggableResources, nil
 }
 
+// GetProjectPodsBySubstr returns a list of pod names that contain the provided string, in the named project.
+func getProjectPodNamesBySubstr(project, substr string) ([]string, error) {
+	ret := []string{}
+	pods, err := GetResourceNames(project, "pod")
+	if err != nil {
+		return nil, err
+	}
+
+	for _, pod := range pods {
+		if strings.Contains(pod, substr) {
+			ret = append(ret, pod)
+		}
+	}
+
+	return ret, nil
```

Comment:
@rhcarvalho here unfortunately I don't care about any language. You can make mess in any language. I take care about QoC and if it's nil/null, we should know why we return it